client/src/components/DatePicker/DatePicker.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import DatePicker from './DatePicker';
+
+function handleClick(e) {
+  e.preventDefault();
+  alert("You clicked the button");
+  //Component called here for searchbar in render
+}
+
+storiesOf('DatePicker', module)
+    .add('DatePicker', () => <DatePicker label='Requested Date*' />)
```

Comment:
We'll need to pass in an `onChange` callback here so that the parent component can control the value of this component

Reply:
Added as function Child with props.onChange.